docs/Doxyfile.in

```diff
@@ -741,7 +741,7 @@ WARN_LOGFILE           =
 # spaces.
 # Note: If this tag is empty the current directory is searched.
 
-INPUT = @abs_top_srcdir@/docs/user_guide.md @abs_top_srcdir@/src
+INPUT = @abs_top_srcdir@/docs/user_guide.md @abs_top_srcdir@/src @config_srcdir@
```

Comment:
What is config_srcdir?

Reply:
It's defined in docs/CMakeLists.txt.  It's the directory to which bvers.f, bufrlib.h and bufrlib.inc are written after they're acted on my configure_file() in cmake, and I wanted them to be included in what gets processed by Doxygen.

Reply:
OK, good! I'll remember this for other NCEPLIBS projects as well. Thanks.